src/utils/queries.js

```diff
@@ -10,4 +10,31 @@ const queryBuilder = (name, target = '', query) => gql`
   }
 `;
 
+const filesQuery = `
```

Comment:
Change this to `.ts` too. Can you put the `gql` prefix? I think it may cause an error but I'm not sure either that should be in here since this is just a string 🤔

Reply:
I didn't understood the gpl prefix part @mcallegari10